src/app/dim-ui/PressTip.tsx

```diff
@@ -259,48 +259,73 @@ export function PressTip(props: Props) {
   const timer = useRef<number>(0);
   const touchStartTime = useRef<number>(0);
   const ref = useRef<HTMLDivElement>(null);
+  const startEvent = useRef<'pointerdown' | 'pointerenter'>();
   const [open, setOpen] = useState<boolean>(false);
 
-  const closeToolTip = useCallback(() => {
+  const closeToolTip = useCallback((e: React.PointerEvent) => {
+    // Ignore events that aren't paired up
+    if (
+      !startEvent.current ||
+      (e.type === 'pointerup' && startEvent.current === 'pointerenter') ||
+      (e.type === 'pointerleave' && startEvent.current === 'pointerdown')
+    ) {
+      return;
+    }
     setOpen(false);
     clearTimeout(timer.current);
-    timer.current = 0;
+    // Don't clear these for 500ms to give the click event time to suppress - pointerup fires before click
+    setTimeout(() => {
+      timer.current = 0;
+      startEvent.current = undefined;
+    }, 500);
```

Comment:
I'm annoyed about having to do this - the other alternative I thought of was to set a "absorb clicks until" timestamp that could be read from `absorbClick` but it all kinda ends up the same way.